x-pack/plugins/reporting/server/routes/generation.ts

```diff
@@ -77,24 +74,6 @@ export function registerJobGenerationRoutes(reporting: ReportingCore, logger: Lo
       });
     }
 
-    if (err instanceof esErrors['401']) {
```

Comment:
Reporting doesn't have access to the legacy client errors, so this code is dead and I removed it. If you considered this logic is necessary, please update it in a follow-up.

Reply:
It looks like this code was not doing anything special but adding a custom message on top of an error code. These error codes seem appropriate: I'm not seeing unexpected transformation of these codes to a custom message. I don't think this logic  was necessary.